tools/SetupFlow/DevHome.SetupFlow/Strings/en-us/Resources.resw

```diff
@@ -1263,4 +1255,97 @@
     <value>Add another account</value>
     <comment>Menu flyout item content to add another account</comment>
   </data>
+  <data name="ConfigurationFieldInvalidType" xml:space="preserve">

```

Comment:
I assume it's not an issue for us to be re(?)-localizing errors from somewhere else?

Reply:
Correct. Only the HRESULT code is communicate over from the API. Error localized messages are up  to the user (in this case Dev Home) to handle.